pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -314,23 +314,30 @@ def __init__(self, *args, **kwargs):
 
         self._autoPart.connect("toggled", self._method_radio_button_toggled)
         self._customPart.connect("toggled", self._method_radio_button_toggled)
-        self._blivetGuiPart.connect("toggled", self._method_radio_button_toggled)
 
         # hide radio buttons for spokes that have been marked as visited by the
         # user interaction config file
         if sam.get_screen_visited("CustomPartitioningSpoke"):
             self._customPart.set_visible(False)
             self._customPart.set_no_show_all(True)
-        if sam.get_screen_visited("BlivetGuiSpoke"):
-            self._blivetGuiPart.set_visible(False)
-            self._blivetGuiPart.set_no_show_all(True)
+
+        blivet_gui_supported = True
```

Comment:
I would prefer to abstract this to a separate method (e.g. `enable_blivet_gui`) and call here that method only.

Reply:
Done